app/_locales/en/messages.json

```diff
@@ -4758,9 +4756,39 @@
   "simulationsSettingSubHeader": {
     "message": "Estimate balance changes"
   },
+  "siweAccount": {
+    "message": "Account"
+  },
+  "siweChainID": {
+    "message": "Chain ID"
+  },
+  "siweIssued": {
+    "message": "Issued"
+  },
+  "siweNetwork": {
+    "message": "Network"
+  },
+  "siweNonce": {
+    "message": "Nonce"
+  },
+  "siweOriginalMessage": {
+    "message": "Message"
+  },
+  "siweRequestId": {
+    "message": "Request Id"
```

Comment:
As with `Chain ID`, should this be `Request ID`?

Reply:
done 👍